src/modules/CurrencyModule.h

```diff
@@ -12,20 +12,20 @@ class CurrencyModule : public Module
 	struct UserCurrency
 	{
 	private:
-		double m_Currency;
+        int m_Currency;
 
 	public:
 		snowflake_t userId;
 		bool isDailyClaimed;
 		bool isBribeUsed;
-		double maxCurrency;
+        int maxCurrency;
 		unsigned int dailyStreak;
 		QTimer* jailTimer;
 		unsigned int numberOfDailysClaimed;
 		unsigned int numberOfGiveawaysClaimed;
 
 		UserCurrency(snowflake_t userId, double currency, double maxCurrency, bool isDailyClaimed, unsigned int dailyStreak, unsigned int numberOfDailysClaimed, unsigned int numberOfGiveawaysClaimed)
-			: userId(userId), maxCurrency(maxCurrency), isDailyClaimed(isDailyClaimed), isBribeUsed(false), dailyStreak(dailyStreak), jailTimer(new QTimer()), numberOfDailysClaimed(numberOfDailysClaimed), numberOfGiveawaysClaimed(numberOfGiveawaysClaimed)
+            : userId(userId), maxCurrency(std::floor(maxCurrency * 100)), isDailyClaimed(isDailyClaimed), isBribeUsed(false), dailyStreak(dailyStreak), jailTimer(new QTimer()), numberOfDailysClaimed(numberOfDailysClaimed), numberOfGiveawaysClaimed(numberOfGiveawaysClaimed)
```

Comment:
Shouldn't this be `maxCurrency(std::floor(maxCurrency / 100.0f))`?

Also might be worth using a proper rounding function instead of `std::floor`